src/postgraphile/http/setupServerSentEvents.ts

```diff
@@ -20,8 +21,8 @@ export default function setupServerSentEvents(req, res, options) {
     req._koaCtx.compress = false
   }
 
-  const sse = str => {
-    if (isKoa) {
+  const sse = (str: string) => {
+    if (isKoa && stream) {
```

Comment:
Ugh. This is one of the things I hate about typed JS. The logic here would be confusing to the reader as they'd be "if it's koa but there is no stream then do res.write? Why would there not be a stream...?" and then read the code to find it's impossible to not have a stream if it's Koa.

Can you use `stream!.write(str)` below instead so that it's easier to read and doesn't change the compiled code?

Reply:
fe907e5